lightning/src/ln/channelmanager.rs

```diff
@@ -173,6 +173,22 @@ struct ClaimableHTLC {
 	onion_payload: OnionPayload,
 }
 
+/// A payment identifier used to correlate an MPP payment's per-path HTLC sources internally.
+#[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
+pub(crate) struct MppId(pub [u8; 32]);
```

Comment:
Given this is only used internally, would an integer be sufficient? It could be tuple'd with the payment hash (or `session_priv`?) for lookups. I may not have a firm understanding of how `session_priv` is used, FWIW. So happy to get a better idea of how is used for each part. 🙂

Reply:
I guess this is pretty much equivalent to a u64 integer? Not sure I follow re: the tuple idea, let me know what `type` of `pending_outbound_payments` you're thinking of